packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -38,6 +38,13 @@ export const nextGCVersion: GCVersion = 4;
  * Otherwise, only enforce GC Tombstone if the passed in value matches the persisted value
  */
 export const gcTombstoneGenerationOptionName = "gcTombstoneGeneration";
+
+/**
+ * This undocumented GC Option (on ContainerRuntime Options) allows an app to enable throwing an error when tombstone
```

Comment:
for my knowledge.. what does "undocumented" mean here?

Reply:
We have a bunch of documented and officaly supported GC options [here](https://github.com/microsoft/FluidFramework/blob/5bcbd3fb5e829971ddaece93730bf3af88b8d041/packages/runtime/container-runtime/src/gc/gcDefinitions.ts#L302).

We have a couple of options that we added as experiments. We start them undocumented and move it to official options if the experiment is successful and we want to make these options permanent.